pylint_django/checkers/models.py

```diff
@@ -35,7 +35,7 @@
     "W%d03"
     % BASE_ID: (
         "Model does not explicitly define __str__ (%s)",
-        "model-no-explicit-unicode",
+        "model-no-explicit-str",
```

Comment:
IMO message-id changes need to be documented in the changelog b/c people rely on them to enable and disable checkers. For someone this will suddenly result in lots of warnings in their pylint output.

Reply:
I did think about that but then I forgot about it; on the one hand the message code should have meaning but as you say it'd mess up a bunch of config. I'm not sure how to handle that.

Reply:
Numeric message codes are deprecated in Pylint. They always tell you to use the human readable ones. IMO a simple conversion table `old-mesage` -> `new-message` in Changelog is enough. 

Another, more complicated option is to keep the old message ids and then add new ones for the same messages to enable a transition period. Not worth it I think.

Reply:
>  I'm not sure how to handle that

Revisiting this PR. Not sure if we can have multiple message codes for the same error.